sql/ddl_api.sql

```diff
@@ -47,69 +47,43 @@ CREATE OR REPLACE FUNCTION  set_number_partitions(
     dimension_name          NAME = NULL
 ) RETURNS VOID AS '@MODULE_PATHNAME@', 'dimension_set_num_slices' LANGUAGE C VOLATILE;
 
--- Drop chunks that are older than a timestamp.
+-- Drop chunks that are older than a timestamp or newer than a timestamp.
 CREATE OR REPLACE FUNCTION drop_chunks(
-    older_than anyelement,
+    older_than ANYELEMENT = NULL,
```

Comment:
Still not clear why we need default arguments on all parameters, given that this seems to indicate that you _can_ call the function without arguments although it fails during execution. So, as I see that the code was changed so that `older_than` now defaults to `NULL` I am wondering if that is strictly necessary and in so case why given that you are probably overriding this anyway with another argument?